pkg/core/block/block_base.go

```diff
@@ -61,6 +66,7 @@ type baseAux struct {
 	Timestamp     uint64                `json:"time"`
 	Index         uint32                `json:"index"`
 	NextConsensus string                `json:"nextconsensus"`
+	PrevStateRoot *util.Uint256         `json:"previousstateroot,omitempty"`
```

Comment:
It's `StateRoot` in the block and `previousstateroot` in JSON, we better use one name everywhere.

Reply:
Fixed.

Reply:
`previous` in JSON to be unified with `previousblockhash`.